iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/StringLiteralImpl.java

```diff
@@ -43,4 +43,9 @@ public String value() {
   public List<Tree> children() {
     return List.of(token);
   }
+
+  @Override
+  public String toString() {
+    return token.toString();
+  }
```

Comment:
Yes, almost every AST element doesn't have one, and it's hard to debug without it (in my sense, at least).
When running unit tests in debug with a breakpoint, you have to expand every dropdown to see which test is currently running, which element corresponds to which part of the test, etc.

It now looks like this:
![image](https://github.com/user-attachments/assets/c66f7e25-910c-4f99-a07f-884f4be19270)


There is no downside to adding `toString`methods. The only one I thought about was with big unit test files. It could generate long strings in debug. But in that case, it fails pretty quickly with an `OutOfMemory`exception; I tested it. I'm trying to add those methods as I need them in my PR since I'm adding them locally anyway :)